test/integration/sandbox-libraries/pm.test.js

```diff
@@ -578,4 +578,322 @@ describe('sandbox library - pm api', function () {
             });
         });
     });
+
+    describe('pm.info properties', function () {
+        describe('pm.info.requestPath', function () {
+            it('should have path as empty array for un-saved requests', function (done) {
+                this.run({
+                    collection: {
+                        item: [{
+                            event: [
+                                {
+                                    listen: 'prerequest',
+                                    script: {
+                                        exec: [
+                                            'console.log("path", pm.info.requestPath)'
+                                        ],
+                                        type: 'text/javascript'
+                                    }
+                                }
+                            ],
+                            request: {
+                                auth: {
+                                    type: 'noauth'
+                                },
+                                method: 'GET',
+                                header: [
+                                    {
+                                        key: 'fs',
+                                        value: '',
+                                        type: 'text'
+                                    }
+                                ],
+                                url: {
+                                    raw: 'postman-echo.com/get',
+                                    host: [
+                                        'postman-echo',
+                                        'com'
+                                    ],
+                                    path: [
+                                        'get'
+                                    ]
+                                }
+                            },
+                            response: []
+                        }]
+                    }
+                }
+                , function (_err, results) {
+                    testrun = results;
+                    expect(testrun).to.be.ok;
+                    sinon.assert.calledOnce(testrun.start);
+                    sinon.assert.calledOnce(testrun.done);
+                    sinon.assert.calledWith(testrun.done.getCall(0), null);
+
+                    const collPreConsole = testrun.console.getCall(0).args.slice(2);
+
+                    expect(collPreConsole).to.deep.include.ordered.members(['path',
+                        []]);
+                    done();
+                });
+            });
+
+
+            describe('saved request cases', function () {
```

Comment:
The suite name should be a noun or an adjective. `describe` with `it` should form a sentence.
e.g.
```javascript
describe('An apple', () => {
  describe('with a worm', () => {
    it('should be thrown away', () => {});
  });
});

// forms a sentence: An apple, with a worm, should be thrown away.
```